src/main/java/org/jitsi/jigasi/transcription/VoskTranscriptionService.java

```diff
@@ -0,0 +1,277 @@
+/*
+ * Jigasi, the JItsi GAteway to SIP.
+ *
+ * Copyright @ 2020 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.jigasi.transcription;
+
+import org.eclipse.jetty.websocket.api.Session;
+import org.eclipse.jetty.websocket.api.annotations.OnWebSocketClose;
+import org.eclipse.jetty.websocket.api.annotations.OnWebSocketConnect;
+import org.eclipse.jetty.websocket.api.annotations.OnWebSocketError;
+import org.eclipse.jetty.websocket.api.annotations.OnWebSocketMessage;
+import org.eclipse.jetty.websocket.client.WebSocketClient;
+import org.jitsi.jigasi.*;
+import org.jitsi.utils.logging.*;
+
+import javax.media.format.*;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.ByteBuffer;
+import java.util.*;
+import java.util.concurrent.CountDownLatch;
+import java.util.function.*;
+
+
+/**
+ * Implements a TranscriptionService which uses local
+ * Vosk websocket transcription service.
+ * <p>
+ * See https://github.com/alphacep/vosk-server for
+ * information about server
+ *
+ * @author Nik Vaessen
+ * @author Damian Minkov
+ * @author Nickolay V. Shmyrev
+ */
+public class VoskTranscriptionService
+        implements TranscriptionService {
+
+    /**
+     * The logger for this class
+     */
+    private final static Logger logger
+            = Logger.getLogger(VoskTranscriptionService.class);
+
+    /**
+     * Property name for username in credential JSON of IBM Watson
+     * speech-to-text service.
+     */
+    public final static String P_URL
```

Comment:
I think the variable name should be a bit more descriptive.

Reply:
Fixed